packages/client/src/store/slices/theme/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface IThemeState {
+  colorScheme: ThemesEnum;
+}
+
+export enum ThemesEnum {
```

Comment:
Более стандартный нейминг для энамов – В единственном числе, и без слова Enum в имени переменной

https://www.typescriptlang.org/docs/handbook/enums.html#string-enums
```suggestion
export enum Theme {
```